libbeat/common/cli/password.go

```diff
@@ -23,7 +23,8 @@ import (
 	"strings"
 	"syscall"
 
```

Comment:
Delete blank line and `go fmt`.

This is the first I noticed, but there are more like this.

Reply:
thanks for the review @efd6 - i have a follow up PR [here](https://github.com/elastic/beats/pull/35827), will make to address similar issues over there.